TwilioVerifyDemoCache/Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.4
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "TwilioVerifyDemoCache",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "TwilioVerifyDemoCache",
+            targets: ["TwilioVerifyDemoCache"]
+        ),
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/64d430897a2df4dbbabee1ff15ea40919450eaae/TwilioVerifyDemoCache/Package.swift#L16"></span>Collection literals should not have trailing commas.<br />
<code>trailing_comma</code> <code>Package.swift:16</code></td>
    </tr>
  </tbody>
</table>